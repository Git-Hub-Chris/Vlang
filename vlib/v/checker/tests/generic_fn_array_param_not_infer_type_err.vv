import arrays

arrays.sum([]) or { panic(err) }
