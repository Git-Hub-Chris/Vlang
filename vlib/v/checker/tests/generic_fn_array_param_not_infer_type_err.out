vlib/v/checker/tests/generic_fn_array_param_not_infer_type_err.vv:3:8: error: could not infer generic type `T` in call to `arrays.sum`
    1 | import arrays
    2 | 
    3 | arrays.sum([]) or { panic(err) }
      |        ~~~~~~~
